Packs/PrismaCloud/TestPlaybooks/Prisma_Cloud_Test.yml

```diff
@@ -1,19 +1,20 @@
 id: Prisma Cloud V2 Test
-version: -1
+version: 14
```

Comment:
Lets revert all changes that are not part of your PR, run format on this file

Reply:
I added tasks for testing access-key flow. it changed ID's and boxes position to the other tasks that were there.